src/modules/esl-panel/core/esl-panel.ts

```diff
@@ -86,32 +102,36 @@ export class ESLPanel extends ESLToggleable {
     });
   }
 
+  /** Process after animation state */
   protected afterAnimate() {
     this.clearAnimation();
     this.$$fire(this.open ? 'after:show' : 'after:hide');
   }
 
+  /** Clear animation */
   protected clearAnimation() {
     this.style.removeProperty('max-height');
     CSSUtil.removeCls(this, this.animateClass);
     CSSUtil.removeCls(this, this.postAnimateClass);
   }
 
+  /** Clear animation after fallback time  */
   protected fallbackAnimate() {
     const time = +this.fallbackDuration;
     if (isNaN(time) || time < 0) return;
     if (this._fallbackTimer) clearTimeout(this._fallbackTimer);
     this._fallbackTimer = window.setTimeout(() => this.afterAnimate(), time);
   }
 
+  /** Clean up the bits of animation */
   @bind
   protected _onTransitionEnd(e?: TransitionEvent) {
     if (!e || e.propertyName === 'max-height') {
       this.afterAnimate();
     }
   }
 
-  /** The panels use panel group config for actions */
+  /** Merge params that are used by panel group for actions */
```

Comment:
Merge params that are used by the panel group for actions

Reply:
updated